idseq_pipeline/commands/host_filtering_functions.py

```diff
@@ -566,8 +614,22 @@ def run_host_filtering(fastq_files, initial_file_type_for_log, lazy_run, stats):
     stats.count_reads("run_bowtie2",
                       before_filename=os.path.join(RESULT_DIR, LZW_OUT1),
                       before_filetype="fasta_paired",
-                      after_filename=os.path.join(RESULT_DIR, EXTRACT_UNMAPPED_FROM_SAM_OUT1),
+                      after_filename=os.path.join(RESULT_DIR, EXTRACT_UNMAPPED_FROM_BOWTIE_SAM_OUT1),
```

Comment:
Please note the function get_remaining_reads() in common.py.   You will have to update it so it now picks up the reads after gsnap not the reads after bowtie2.
```
    def get_remaining_reads(self):
        return (item for item in self.data if item.get("task") == "run_bowtie2").next().get("reads_after")
```

Reply:
Or whichever is available, I guess.

Reply:
OK. interesting. why does it need it in pipeline?